packages/db/fauna/resources/Function/createUpload.js

```diff
@@ -103,6 +103,20 @@ const body = Query(
                         )
                       )
                     ),
+                    Foreach(
+                      Select('backupData', Var('data')),
+                      Lambda(
+                        ['data'],
+                        Create('Backup', {
```

Comment:
If a user uploads the same file twice then we'll get multiple backup objects for the same CAR pointing to the same bucket+key?

Reply:
We will get a single S3 object stored considering it is stored on a key value fashion, but multiple entries in Fauna for the same file.

I thought about adding an index at first, but it would run in every single upload to check if it already exists. Using `@unique` would also fail, which we don't want.

So, given we should just iterate on the list of objects in S3 for backup I think the best solution is to simply keep record of everything as is. With the record, we can easily access specific data to prfioritize backups as needed.

What do you think?

Reply:
Regarding this, If a same upload is created, we will not create the Upload entry in Fauna, which means we will not create the backups for this in Fauna as well.

Reply:
The above is only true because there is a bug 🤦🏼  We are not adding the following chunks to Fauna... Working on a PR